lib/registries/addon/components/sharing-icons/component.ts

```diff
@@ -0,0 +1,65 @@
+import { computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import Analytics from 'ember-osf-web/services/analytics';
+import $ from 'jquery';
```

Comment:
We're hoping to one day remove jquery from our builds. We have our own [param function](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/utils/param.ts) to replace `$.param`

Reply:
Neat!